src/PrestaShopBundle/Form/Admin/Sell/Catalog/AttributeType.php

```diff
@@ -0,0 +1,146 @@
+
+use PrestaShop\PrestaShop\Adapter\AttributeGroup\AttributeGroupDataProvider;
+use PrestaShop\PrestaShop\Adapter\AttributeGroup\Repository\AttributeGroupRepository;
+use PrestaShop\PrestaShop\Core\ConstraintValidator\Constraints\TypedRegex;
+use PrestaShop\PrestaShop\Core\ConstraintValidator\TypedRegexValidator;
+use PrestaShop\PrestaShop\Core\Domain\AttributeGroup\ValueObject\AttributeGroupId;
+use PrestaShop\PrestaShop\Core\Domain\AttributeGroup\ValueObject\AttributeGroupType;
+use PrestaShop\PrestaShop\Core\Domain\Language\ValueObject\LanguageId;
+use PrestaShop\PrestaShop\Core\Domain\Shop\ValueObject\ShopId;
+use PrestaShopBundle\Form\Admin\Type\ShopChoiceTreeType;
+use PrestaShopBundle\Form\Admin\Type\TranslatableType;
+use PrestaShopBundle\Form\Admin\Type\TranslatorAwareType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\ColorType;
+use Symfony\Component\Form\Extension\Core\Type\FileType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Validator\Constraints\NotBlank;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+/**
+ * Form type for attribute add/edit
+ */
+class AttributeType extends TranslatorAwareType
+{
+    /**
+     * @var AttributeGroupRepository
+     */
+    private $attributeGroupRepository;
+
+    /**
+     * @var AttributeGroupDataProvider
+     */
+    private $attributeGroupDataProvider;
+
+    /**
+     * @var int
+     */
+    private $contextShopId;
+
+    /**
+     * @var int
+     */
+    private $contextLangId;
+
+    public function __construct(
+        TranslatorInterface $translator,
+        array $locales,
+        AttributeGroupRepository $attributeGroupRepository,
+        AttributeGroupDataProvider $attributeGroupDataProvider,
+        int $contextShopId,
+        int $contextLangId
+    ) {
+        parent::__construct($translator, $locales);
+
+        $this->attributeGroupRepository = $attributeGroupRepository;
+        $this->contextShopId = $contextShopId;
+        $this->attributeGroupDataProvider = $attributeGroupDataProvider;
+        $this->contextLangId = $contextLangId;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $attributeGroupId = $options['data']['attribute_group'];
+        $attributeGroup = $this->attributeGroupRepository->get(
+            new AttributeGroupId($attributeGroupId),
+            new ShopId($this->contextShopId)
+        );
+        $builder
+            ->add('attribute_group', ChoiceType::class, [
+                'label' => $this->trans('Attribute group', 'Admin.Catalog.Feature'),
+                'help' => $this->trans('The way the attribute\'s values will be presented to the customers in the product\'s page.', 'Admin.Catalog.Help'),
+                'choices' => $this->attributeGroupDataProvider->getAttributeGroupChoices(
```

Comment:
As comment above, this could be handled completely internally but this form type itself